docs/api/zniffer.md

```diff
@@ -0,0 +1,347 @@
+# Zniffer
+
+A Zniffer is a Z-Wave controller with special firmware that can capture Z-Wave traffic from any nearby Z-Wave device. This is useful for troubleshooting, as it allows you to see exactly what is being sent and received by the devices in your network.
+
```

Comment:
I suspect many users will start to pay more attention to this area, as such, maybe add some brief outline on how to obtain the FW (i.e links) and flash a sniffer to the stick?